addon/components/frost-chart-y-axis.js

```diff
@@ -118,15 +119,21 @@ export default Component.extend({
 
   init () {
     this._super(...arguments)
-    this.attrs.dispatch({
+    this.dispatch({
       type: 'REGISTER_AXIS'
     })
   },
 
-  didInsertElement () {
-    this._super(...arguments)
-    run.scheduleOnce('afterRender', this, this._dispatchRenderedAxis)
-  }
-
   // == Actions ===============================================================
+  actions: {
+    _dispatchTickRendered (action) {
```

Comment:
That's my point @dafortin - they are always internal because that's the Ember way of things.  We do not need an underscore the beleaguer this point.